src/index.js

```diff
@@ -299,13 +300,33 @@ export default class AuthnWidget {
   }
 
   postDeviceProfileAction() {
-    let profilingElement = document.querySelector('[data-profilingtype]');
-    switch (profilingElement.dataset['profilingtype']) {
+    let data = this.store.getStore();
+    switch (data.deviceProfilingType) {
       case 'IDW':
+      case 'TMX-WEB':
+        var script = document.createElement('script');
```

Comment:
Does the script do anything in this scenario? Doesn't seem like it will get executed

Reply:
The script will look like
`<script src="https://${deviceDomain}/fp/tags.js?org_id=${orgId}&session_id=${sessionId}"></script> ` and it gets executed automatically.

Reply:
I see, so is IDW supposed to be doing this same thing?

Reply:
IDW did the same thing before (well it was suppose to, I originally called the script for IDW in handle bar but later realized that script in handle bar doesn't work so I moved everything here)